controls/include/controls/six_deg_PID.h

```diff
@@ -0,0 +1,44 @@
+//
+// Created by tanvi on 11/21/2020.
+//
+
+#ifndef CONTROLS_SIX_DEG_PID_H
+#define CONTROLS_SIX_DEG_PID_H
+#include "PID.h"
+#include <vector>
+
+class six_deg_PID

```

Comment:
Class names should be all caps, no underscores. So, `SixDegPID`. Also, I would generally avoid abbreviations as it is not that much harder to type out (esp. with tab complete) and much easier to read. So, I would rename this class to `SixDegreePID`